app/scripts/models/user.js

```diff
@@ -158,9 +180,11 @@ define([
       var account = self.initAccount(accountData);
       account.set('lastLogin', Date.now());
 
+      self._cachedSignedInAccount = account;
+
       return self.setAccount(account)
```

Comment:
Can `setAccount` fail here and then we'd have a `_cachedSignedInAccount` that may be out of whack with reality?